web/src/components/itemList.js

```diff
@@ -68,10 +73,17 @@ class ItemList extends Component {
                         <Button variant="link" style={{padding: '0'}} id={i} onClick={this.currencyConvert}>{this.state.currency[i] ? 'Convert To KRW' : 'Convert To USD'}</Button>
                     </Card.Text>
                     <Button variant={
-                        this.selected[i] ? 'secondary' : 'primary'
-                    } id={i} onClick={this.itemClicked}>{
-                        this.selected[i] ? 'Remove from cart' : 'Add to cart'
+                        'primary'
+                    } id={i} onClick={this.addItemClicked}>{
+                        'Add to cart'
+                    }</Button>
+                    <Button variant={
+                        'secondary'
+                    } id={i} onClick={this.removeItemClicked}>{
+                        'Remove from cart'
                     }</Button>
+                    <br></br>
+                    개수: {this.selected[i]}
```

Comment:
If you want to update the shape of the component, you should use state other than a general object.